specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManager.json

```diff
@@ -437,6 +437,218 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/connectToScope": {
+      "get": {
```

Comment:
There are a number of issues with this PR. please fix before submitting for ARM review.

- Please add examples for the new apis
- This seems like it has some breaking changes in the same api version. 
- The JSON is incorrectly formatted and does not render on swagger editors.